cmd/snap/cmd_create_user.go

```diff
@@ -36,6 +37,8 @@ An account can be setup at https://login.ubuntu.com.
 `)
 
 type cmdCreateUser struct {
+	Yaml       bool `long:"yaml" description:"output results in YAML format"`
```

Comment:
Yeah, I think console-conf using the api directly might well make sense (and not really being any harder). Do this for now?

Changed to JSON though.